static/connectors.json

```diff
@@ -5207,6 +5207,26 @@
       }
     ]
   },
+  {
+    "nameWithOwner": "ConduitIO/conduit-operator",
```

Comment:
We should write a changelog for htis.

Reply:
https://github.com/ConduitIO/conduit-site/issues/171

Reply:
Yes, but we should also add this to the ignore list, as the operator is not a connector and we don't want it in the list.

Reply:
@lovromazgon good catch.